### ISSUE TITLE
Buy credits with commerce

### ISSUE THREAD

**Issue by johanneslamers** [madebyshape/points#2]
Is it possible to buy credits with commerce or stripe? Coule be an awesome feature.

**Comment by madebyshape** [comment]
I did consider this, to work with Craft Commerce. Need to play with Craft Commerce a bit more to add this functionality in :)

**Comment by neilbradley** [comment]
Integrating with Commerce would be great to build a Loyalty programme for each product purchased and then to be able to use those credits to apply a discount on future purchases.

**Comment by bymayo** [comment]
It's in the works :) Just a bit busy at the moment to rework it for Commerce. Might even be a Craft 3 version only. We'll see.

**Comment by adamskyle** [comment]
@bymayo Any news on this rework? We are considering using the current version as a starting point for a simple loyalty implementation but I'm curious when we can expect a new release.

**Comment by bymayo** [comment]
@kyle51north Yeah, it's on the list to work on... BUT - With the release of Craft 3 (And Craft Commerce 2 Beta at beginning of 2018) this plugin won't be reworked with this additional functionality for Craft 2. 

I haven't got time to add the functionality in for Craft 2, then rework it for Craft 3 sadly 👎 

Is it likely you'll be using Craft 3?

**Comment by adamskyle** [comment]
@bymayo We've decided to roll out the project in two phases, first with Craft 2 end January and them with more functionalities after Craft 3 release. So yes, we'll be using Craft 3, but at a later stage. Will be sure to check out this repository!

**Comment by bymayo** [comment]
@johanneslamers @neilbradley @adamskyle I know this was an issue from 10 years ago 🙈  Wow, this is bad even my issue fixing standards 😂

But i've just brought this plugin back from the dead for Craft 5 and this feature is now available in 5.0.0.

Sorry for the er, wait 😂

Give it a go!
